src/docs/concepts-listings/how-the-search-works/index.md

```diff
@@ -12,10 +12,45 @@ published: true
 
 ## Listings search in Flex
 
-In Flex listings are searched by using the
+Flex has a powerful listing search engine, which can find listings based
+on multiple criteria:
+
+- **Geolocation.** The search can be used to display only listings that
+  are within a provided radius from certain coordinates.
+- **Free text.** The search can be used to find listings that have a
+  certain keyword provided by the user. The search can find from listing
+  title and description. You can also choose to index some public data
+  fields (like listing category) so the search finds from them as well.
+- **Price.** It's possible to filter out listings with too high or too
+  low price.
+- **Availability.** It's possible to filter out listings that have
```

Comment:
It would be cool to have an example query of each using the SDK!

Reply:
Good idea! I added examples to the ones that aren't dealt with more deeply in the second half of the article